mantidproject/mantid
# Assign resolution properly on bin boundaries

This PR deals with two issues:
 1- The Q resolution values were not assigned properly, according to #16119.
     I changed my mind on this one. Playing around with the values at bin boundaries creates artifacts in the Dx array. It's preferable to leave it as it is so that both the workspace and the saved I(q) file can safely be used. The original problem only affects the last point of the distribution, only when you move to point data. That point has a dQ that is underestimated. It's best to deal with this in the refactor mentioned in #16119.
 2- The instrument scientists would like to update the EQSANS TOF resolution function.

**To test:**
- Make sure the system tests pass.
- Inspect the code.

Fixes #16119.

No release notes needed.

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
